bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html

```diff
@@ -6,26 +6,28 @@
 
 <footer class="platform-footer">
   <nav class="platform-footer__navigation">
-    <ul class="platform-footer__navigation-links">
-      <li class="platform-footer__navigation-links-item"><a>Blog</a></li>
-      <li class="platform-footer__navigation-links-item"><a>About</a></li>
-      <li class="platform-footer__navigation-links-item"><a>Explore</a></li>
-      <li class="platform-footer__navigation-links-item"><a>Publishers</a></li>
-      <li class="platform-footer__navigation-links-item"><a>Developers</a></li>
-      <li class="platform-footer__navigation-links-item"><a>Terms of Service</a></li>
-      <li class="platform-footer__navigation-links-item"><a>Privacy</a></li>
-      <li class="platform-footer__navigation-links-item"><a>Support</a></li>
-      <li class="platform-footer__navigation-links-item"><a>Jobs</a></li>
+    <ul class="platform-footer__navigation--list">
+      <li class="platform-footer__navigation--list-item"><a>Blog</a></li>
+      <li class="platform-footer__navigation--list-item"><a>About</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Explore</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Publishers</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Developers</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Terms of Service</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Privacy</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Support</a></li>
+      <li class="platform-footer__navigation--list-item"><a>Jobs</a></li>
     </ul>
-    <div class="platform-footer__navigation-social-media">
-      <ul class="platform-footer__navigation-social-media-list">
-        <li class="platform-footer__navigation-social-media-list-text">Connect:</li>
-        <li class="platform-footer__navigation-social-media-list-item twitter"><a href="https://twitter.com/pocket" target="_blank" rel="noreferrer">Twitter</a></li>
-        <li class="platform-footer__navigation-social-media-list-item facebook"><a href="https://facebook.com/pocket" target="_blank" rel="noreferrer">Facebook</a></li>
-        <li class="platform-footer__navigation-social-media-list-item google"><a target="_blank" rel="noreferrer">Google Plus</a></li>
+    <div class="platform-footer__navigation--social-media">
+      <ul class="platform-footer__navigation--social-media--list">
+        <li class="platform-footer__navigation--social-media--list-text">Connect:</li>
```

Comment:
This should be a `span` element because it's not really considered a list item, so it doesn't make sense to put it in `li` tags
```suggestion
        <li class="platform-footer__navigation--social-media--list-text">Connect:</li>
```
```suggestion
        <span class="platform-footer__navigation--social-media--list-text">Connect:</span>
```